RNReanimated.podspec

```diff
@@ -54,12 +54,16 @@ end
 
 rnVersion = reactVersion.split('.')[1]
 
+fabric_enabled = ENV['RCT_NEW_ARCH_ENABLED'] == '1'
+if fabric_enabled
+  raise "[Reanimated] Reanimated 2.x does not support Fabric. Please upgrade to react-native-reanimated@3.0.0-rc.2 to enable Fabric support. For details, see https://blog.swmansion.com/announcing-reanimated-3-16167428c5f7"
```

Comment:
I wouldn't mention a particular version of RC in the error as it works with previous (and future I hope!) RC's too.  Maybe `react-native-reanimated@^3` would do the trick? Or just Reanimated 3.x as in the previous sentence?